### PR TITLE
deployer should exit with error status code on exception

### PR DESCRIPTION
**Pull request by fliespl**
Currently if exception happens inside deploy.php CI will mark job as succeeded.

This PR adds exit code to run command in case it fails with an exception.

- [X] Bug fix #…?
- [ ] New feature?
- [ ] BC breaks?
- [ ] Tests added?
- [ ] Docs added?

### CODE REVIEW

File: src/Deployer.php [hunk 1]
```diff
@@ -315,6 +315,8 @@ public static function run(string $version, ?string $deployFile)
                 $output->setVerbosity(OutputInterface::VERBOSITY_DEBUG);
             }
             self::printException($output, $exception);
+            
+            exit(1);
```
Comment (antonmedv) [inline src/Deployer.php:319]: Let’s use Symfony h onds fir this.

Comment (antonmedv) [reply src/Deployer.php:319]: Methods

**Review by antonmedv — CHANGES_REQUESTED** [review]

File: src/Deployer.php (hunk 1 above)
Comment (fliespl) [reply src/Deployer.php:319]: Do you mean Command constants? Cause your current console implementation can't rely on "returning" status code.

Comment (antonmedv) [reply src/Deployer.php:319]: Oh, I see.

**Review by antonmedv — APPROVED** [review]

**Pull request merged**